modules/ROOT/pages/_partials/common-on-new-file.adoc

```diff
@@ -1,12 +1,12 @@
-The File, FTP, and SFTP connectors provide a listener (called `On New or Updated file` in Studio and Flow Designer UI) that polls a directory for files that have been created or updated. One message is generated for each file that is found.
+The File, FTP, and SFTP connectors provide a listener (called `On New or Updated file` in Studio and in the Flow Designer user interface) that polls a directory for files that have been created or updated. A message is generated for each file that is found.
```

Comment:
"Flow Designer" -> "Design Center" ? I don't think we're supposed to use FD. It's not in the UI.

Reply:
Changed to:
The File, FTP, and SFTP connectors provide a listener (called `On New or Updated file` in Studio and Design Center) that polls a directory for files that have been created or updated.